src/GDGUkraine/templates/event.html

```diff
@@ -9,6 +9,19 @@
 	<meta name="author" content="Olexandr Kovalchuk">
 	<meta name="author" content="Svyatoslav Sydorenko">
 	<meta name="author" content="Valentyn Shybanov">
+    <meta name="author" content="{{ event.host_gdg.name or 'GDG Ukraine' if event.host_gdg_id else 'GDG Ukraine'}}"/>
```

Comment:
Yes, of course it does)
http://jinja.pocoo.org/docs/dev/templates/#if-expression